tests/robustness/validate/operations.go

```diff
@@ -53,6 +54,8 @@ func validateLinearizableOperationsAndVisualize(
 	operations []porcupine.Operation,
 	timeout time.Duration,
 ) (results Results) {
+	operations = patchResponseTimeOfFailedOperations(operations)
```

Comment:
This should happen before we patch operations, because then patching will not have opportunity to modify the return time.